### PR TITLE
Feature/46308

### PR DESCRIPTION
**Pull request by Aekal**
dev-46308
#380

### CODE REVIEW

File: components/03-modules/active-filters/active-filters.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
       {
         label: 'Label',
         value: 'Value',
-        ariaLabel: 'Disable filter',
+        ariaLabel: 'Disable filter: Value',
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/active-filters/active-filters.config.js]: Added

File: components/03-modules/filter/filter--color.hbs [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
-                </a>
-            {{/each}}
-        </div>
+        <ul class="list filter__list">
+            {{#each options }}
+                <li class="filter__color-item">
+                    <a
+                        href='{{ href }}'
+                        class="filter__color-option {{ class }}"
+                        aria-label="{{ ariaLabel }}"
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/filter/filter--color.hbs]: Not really, removed ;)

File: components/03-modules/filter/filter--swatch.hbs [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
-                   aria-label="{{ ariaLabel }}"
-                   {{{ attributes }}}
-                >
-                    {{ text }}
-                </a>
+                <li class="filter__swatch-item">
+                    <a
+                        href="{{ href }}"
+                        class="filter__swatch-option {{ class }}"
+                        aria-label="{{ ariaLabel }}"
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/03-modules/filter/filter--swatch.hbs]: Removed

File: components/03-modules/filter/filter--color.hbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (anqaka) [reply components/03-modules/filter/filter--color.hbs]: 👍

File: components/03-modules/filter/filter--swatch.hbs (hunk 3 above)
[the remark this answers is not in the document]
Comment (anqaka) [reply components/03-modules/filter/filter--swatch.hbs]: 👍

**Comment by anqaka** [comment]
Good one!

**Pull request merged**